src/Entities/Player.cpp

```diff
@@ -2023,11 +2050,12 @@ bool cPlayer::DoMoveToWorld(cWorld * a_World, bool a_ShouldSendRespawn, Vector3d
 		}
 
 		// Broadcast the player into the new world.
+		a_World->BroadcastPlayerListAddPlayer(*this);
```

Comment:
I don't think this is right, the player should already be on the list.  Would `BroadcastPlayerListUpdateGameMode` be enough?

Reply:
After test, BroadcastPlayerListAddPlayer or BroadcastPlayerListUpdateGameMode is not needed when move player to other world.

I had some misunderstand about protocol wiki...